### PR TITLE
Add missing snapshots - part 2

### PR DESCRIPTION
**Pull request by EmilHvitfeldt**
This is the second half of https://github.com/tidymodels/recipes/pull/1383.

i added snapshots for the rest of the errors. there was 5 errors i was not able to trigger. they have all been labeled as such for future us

### CODE REVIEW

File: tests/testthat/_snaps/tune_args.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    Code
+      recipe(~., data = mtcars) %>% step_pca(all_predictors(), num_comp = ~ tune() +
+        tune()) %>% tune_args()
+    Condition
+      Error in `purrr::map()`:
+      i In index: 1.
+      Caused by error in `purrr::map_chr()`:
+      i In index: 1.
+      i With name: num_comp.
```
Comment (simonpcouch) [inline tests/testthat/_snaps/tune_args.md:11]: I thiiink this is the same place as what I called out before, but would be awesome to trim these two layers of rethrowing out of the message.

**Review by simonpcouch — APPROVED** [review]
VERY VERY GROOVY🪩

File: tests/testthat/_snaps/extract_parameter.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# rethrows error correctly from implementation
+
+    Code
+      params <- extract_parameter_set_dials(rec)
+    Condition
+      Error in `mutate()`:
+      i In argument: `object = purrr::map(call_info, eval_call_info)`.
+      Caused by error in `purrr::map()`:
+      i In index: 1.
+      Caused by error in `.f()`:
```
[the remark this answers is not in the document]
Comment (EmilHvitfeldt) [reply tests/testthat/_snaps/extract_parameter.md]: fixed in [`b6fd10b` (#1388)](https://github.com/tidymodels/recipes/pull/1388/commits/b6fd10b36a789826625e1e64f656e7fe84aa0ae9)

File: tests/testthat/_snaps/tune_args.md (hunk 1 above)
Comment (EmilHvitfeldt) [reply tests/testthat/_snaps/tune_args.md]: fixed in [`b6fd10b` (#1388)](https://github.com/tidymodels/recipes/pull/1388/commits/b6fd10b36a789826625e1e64f656e7fe84aa0ae9)

**Pull request merged**